content/en/real_user_monitoring/mobile_and_tv_monitoring/integrated_libraries/ios.md

```diff
@@ -0,0 +1,52 @@
+---
+title: iOS and tvOS Libraries for RUM
+code_lang: ios
+type: multi-code-lang
+code_lang_weight: 10
+aliases:
+- /real_user_monitoring/ios/integrated_libraries/
+further_reading:
+- link: https://github.com/DataDog/dd-sdk-ios
+  tag: "Source Code"
+  text: Source code for dd-sdk-ios
+- link: /real_user_monitoring/mobile_and_tv_monitoring/integrated_libraries
+  tag: Documentation
+  text: Integrated Libraries
+---
+
+This page lists integrated libraries you can use for iOS and tvOS applications.
+
+## Alamofire
+
+Starting from version `2.5.0`, the RUM iOS SDK can automatically track [Alamofire][1] requests.
+
+1. Configure RUM monitoring by following the [Setup][2] guide.
+2. Enable `URLSessionInstrumentation` for `Alamofire.SessionDelegate`:
+
+```swift
+import Alamofire
+import DatadogRUM
+
+URLSessionInstrumentation.enable(with: .init(delegateClass: SessionDelegate.self))
+```
+For additional information on sampling rate, distributed tracing, and adding custom attributes to tracked RUM resources, refer to "Automatically track network requests" in the [Advanced Configuration][4] documentation.
```

Comment:
```suggestion
`SessionDelegate` is a delegate class implemented in the target application.

For additional information on sampling rate, distributed tracing, and adding custom attributes to tracked RUM resources, refer to "Automatically track network requests" in the [Advanced Configuration][4] documentation.
```

Reply:
This isn't correct. Either we state `Alamofire.SessionDelegate` in the snippet or we add a link in 2nd point to reference [`Alamofire.SessionDelegate`](https://github.com/Alamofire/Alamofire/blob/master/Documentation/AdvancedUsage.md#sessiondelegate) or we do both. WDYT?